getting-started/migrating-data.md

```diff
@@ -1,9 +1,9 @@
 # Migrating Data
 >:TIP: First make sure that you have properly [installed][] **AND [setup][]** your Timescale database within your PostgreSQL instance.
 
-There are two choices available to migrate data into TimescaleDB:
+There are three choices available to migrate data into TimescaleDB:
```

Comment:
How about "There are a number of choices available for migrating data into TimescaleDB:"

(Otherwise, "three" seems *limiting*.)